src/Core/Core.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.0;netstandard2.0</TargetFrameworks>
```

Comment:
like it is not required

Reply:
If it is for reference to `System.Text.Json` then I would prefer to move dependency in `Transport.AspNetCore` project.